lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -166,6 +168,21 @@ def create(self, subsection, course_structure, course):
             self._compute_and_save_grade(subsection, course_structure, course)
         )
 
+    def update(self, usage_key, course_key):
+        """
+        Updates the SubsectionGrade object for the student and subsection.
```

Comment:
Nit: Either update the docstring to say "subsection identified by the given usage_key" or rename `usage_key` to `subsection_usage_key`.